src/commands/install.yml

```diff
@@ -88,6 +88,30 @@ steps:
                   exit 1
               esac
 
+              # Install `less` for paging if not present
+
+              if [ -z "$(command -v less)" ]; then
+                case $SYS_ENV_PLATFORM in
+                  linux)
+                    if [ -x "$(command -v apt-get)" ]; then
+                      $SUDO apt-get install less
+                    else
+                      echo "Unable to install missing less command on this platform."
+                    fi
+                    ;;
+                  *)
```

Comment:
I don't know anything about MacOS / Darwin, so if there is an easy way of having this installed here, I am happy to add it.

Reply:
I found that our macOS executors already have `less` installed so I don't think we need to do it here 🎉